test/lib/slim-rails_assets_test.rb

```diff
@@ -28,6 +28,6 @@ class DummyApp < ::Rails::Application
 
   test 'compile slim view' do
     assert_equal 'ok', with_app(false, 'print DummyApp.assets || "ok"')
-    assert_equal '<div class="test">hi</div>', with_app(true, 'print DummyApp.assets["test.slim"].to_s')
```

Comment:
@januszm 
The former assertion checks whether `test.slim` is generated by the precompile.
(Actually as I mentioned the PR description, the file is generated. so this assertion passed.)
However we expect `test.html` is generated, so I fixed this line.